twins/network.go

```diff
@@ -143,7 +143,7 @@ func (n *Network) createTwinsNodes(nodes []NodeID, _ Scenario, consensusName str
 			consensus.New(consensusModule),
 			consensus.NewVotingMachine(),
 			crypto.NewCache(ecdsa.New(), 100),
-			synchronizer.New(FixedTimeout(0)),
+			synchronizer.New(FixedTimeout(1*time.Millisecond)),
```

Comment:
We should consider if it would be enough to use e.g., `100 * time.Microsecond` to avoid that twins tests take longer time. This should be tested and maybe "measured" since I'm not sure this will have an impact, one way or the other.

Reply:
I tried with different values, it looks like 1 millisecond is more stable compared to lower durations.